shipit-engine.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.rdoc"]
   s.test_files = Dir["test/**/*"]
 
-  s.add_dependency 'rails', '~> 4.2.2'
+  s.add_dependency 'rails', '~> 4.2.0'
```

Comment:
> I ran into a conflict because I ran rails new with Rails 4.2.1

Oh so you didn't use the template? https://github.com/Shopify/shipit-engine/blob/master/template.rb